Rest/Controller/ContentController.php

```diff
@@ -238,6 +254,26 @@ protected function prepareFields(ContentType $contentType, $fields = null)
         return $fields;
     }
 
+    /**
+     * Returns parameters from Request query specified in $allowedParameters.
+     *
+     * @param Request $request
+     * @param array $allowedParameters
+     * @return ParameterBag
+     */
+    protected function parseParameters(Request $request, array $allowedParameters)
```

Comment:
Is there really a reason for such filtering? You should just get what you need from request query.
*(unless you're reusing the whole parameter bag later on somewhere or I'm missing something else :) )*

Reply:
It could be hard to create single router and configure all possible optional parameters like:
`path, hidden, image, siteaccess, webHook, transaction, fields ..`,  the result url could look like `baseurl/null/0/x/name` or `baseurl/path/null/hidden/0/image/x/siteaccess/name` - It's simpler to use query string and provide only needed parameters.